README.md

```diff
@@ -39,19 +71,24 @@ configen --plist-path <plist> --hints-path <mapping-file> --class-name <output-c
 
 # e.g.
 
-configen --plist-path EnvironmentConfig/EnvironmentConfig.plist --hints-path EnvironmentConfig.map --class-name EnvironmentConfig --output-directory EnvironmentConfig
+configen --plist-path EnvironmentConfig/EnvironmentConfig_Prod.plist --hints-path EnvironmentConfig.map --class-name EnvironmentConfig --output-directory EnvironmentConfig
 
 ```
 
 `configen` generates Swift files by default. However, you can generate Objective-C files by providing `objc` as the final argument.
```

Comment:
you can remove this line now. it's no longer true, since we moved to using `--flags`. Maybe replace it with:

> `conigen` generates Swift code by default. You can generate Objective-C code by providing the `-c` or `--objective-c` switches